src/Sinch/Voice/Hooks/AnsweredCallEvent.cs

```diff
@@ -64,5 +63,7 @@ public class AnsweredCallEvent
         /// </summary>
         [JsonPropertyName("amd")]
         public Amd? Amd { get; set; }
+
+        public EventType? EventType { get; }
```

Comment:
Seems this field is not coming present from specs. Could be removed?

Reply:
⚠️ This PR was merged, but this field is still defined

Reply:
Okay, re checked  - seems like false alarm @JPPortier